src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
-# Changelog
+# Changelog - OpenTelemetry.Exporter.Geneva
 
-## Unreleased
```

Comment:
Don't remove the `## Unreleased` section. We always have it at the top of the CHANGELOG